assets/src/design-system/components/input/index.js

```diff
@@ -39,13 +41,32 @@ const Hint = styled(Text)`
   margin-top: 12px;
 `;
 
+const SuffixContainer = styled.div`
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  right: 4px;
+  top: 38px;
+  height: 32px;
+  width: 32px;
+  background: ${({ theme }) => theme.colors.bg.primary};
+  color: ${({ theme }) => theme.colors.fg.tertiary};
+  pointer-events: none;
+  overflow: hidden;
+
+  * {
+    color: ${({ theme }) => theme.colors.fg.tertiary};
```

Comment:
Can't this one just be set to `inherit`, and then you can drop overwriting it below a few times?